ui/app/routes/vault/cluster/secrets/backend/configuration/edit.ts

```diff
@@ -91,6 +106,8 @@ export default class SecretsBackendConfigurationEdit extends Route {
         model['identity-oidc-config'] = { queryIssuerError: true };
       }
     }
+    // add displayName to the model
+    model['displayName'] = displayName;
```

Comment:
This saves us a little code as we use displayName at least twice in the edit.hbs file. Instead of calling the method to return displayName, now I can just access it on the model. It's a very small savings, and if this annoys folks, I'm happy to just call the method in its place.